app.py

```diff
@@ -2070,6 +2070,85 @@ def update_project_status():
+    cursor = conn.cursor()
+
+    if request.method == 'POST':
+        # Handle form submission and update all the project description fields
+        about_organisation = request.form.get('about_organisation')
+        duties_placement = request.form.get('duties_placement')
+        skills_prerequisites = request.form.get('skills_prerequisites')
+        benefits_students = request.form.get('benefits_students')
+        position_title = request.form.get('position_title')
+        position_description = request.form.get('position_description')
+        skill_requirements = request.form.get('skill_requirements')
+        about_org_word_limit = request.form.get('about_org_word_limit', 70)
+        position_desc_word_min = request.form.get('position_desc_word_min', 400)
+        position_desc_word_max = request.form.get('position_desc_word_max', 500)
+        skills_word_min = request.form.get('skills_word_min', 50)
+        skills_word_max = request.form.get('skills_word_max', 70)
+
+        # Get current date for last edit
+        from datetime import datetime
+        current_date = datetime.now().strftime('%d/%m/%Y')
+
+        try:
+            # Update the project record for the current project
+            cursor.execute('''UPDATE Projects 
+                             SET organization_description = ?, 
+                                 project_description = ?, 
+                                 skill_requirements = ?,
+                                 duties_placement = ?,
+                                 skills_prerequisites = ?,
+                                 benefits_students = ?,
+                                 about_organisation = ?,
+                                 position_title = ?,
+                                 position_description = ?,
+                                 key_skills_development = ?,
+                                 about_org_word_limit = ?,
+                                 about_org_last_edit_date = ?,
+                                 position_desc_word_min = ?,
+                                 position_desc_word_max = ?,
+                                 position_desc_last_edit_date = ?,
+                                 skills_word_min = ?,
+                                 skills_last_edit_date = ?,
+                                 skills_word_max = ?
+                             WHERE id = ?''', 
+                          (about_organisation, position_description, skill_requirements, 
+                           duties_placement, skills_prerequisites, benefits_students,
+                           about_organisation, position_title, position_description, 
+                           skill_requirements, about_org_word_limit, current_date,
+                           position_desc_word_min, position_desc_word_max, current_date,
+                           skills_word_min, current_date, skills_word_max, project_id))
+
+            # Update all word limits for all projects
```

Comment:
I don't understand why there are three SQL updates on the same project table here?